### ISSUE TITLE
Accessibility Testing for Veteran Facing Forms - 20-10207 Priority Processing

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/va.gov-team#76572]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@jeana-adhoc 

### Use of color and color contrast checks

- [X] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

No issues

### Who completed the axe scans?

@jeana-adhoc 

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

| Axe Results | Page |
| -- | -- |
| No Issues | [Intro](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/introduction) |
| No Issues  | [3rd party identity - Veteran](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/third-party-veteran-identity) |
| No Issues  | [3rd party identity - claimant](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/third-party-non-veteran-identity) |
| No Issues  | [Your identity](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/preparer-type) |
| No Issues   | [Your personal information](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/veteran-name-and-date-of-birth-a) |
| No Issues  | [Veteran's personal info (Claimant flow)](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/veteran-name-and-date-of-birth-b) |
| No Issues  | [Claimant personal info](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/non-veteran-name-and-date-of-birth) |
| No Issues   | [Identity information](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/veteran-identification-information-a) |
| No Issues  | [Claimant identity information](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/non-veteran-identification-information) | 
| No Issues  | [Veteran identity info Claimant flow](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/veteran-identification-information-b)
| No Issues  | [Housing situation](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/living-situation) |
| No Issues  | [Housing situation - 3rd party Veteran](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/living-situation-third-party-veteran) | 
| No Issues  | [Housing situation - 3rd party claimant](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/living-situation-third-party-non-veteran) |
| No Issues  | [Other housing risk](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/other-housing-risks) |
| No Issues  | [Do you have a mailing address](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/mailing-address-yes-no) |
| No Issues  | [Mailing address](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/veteran-mailing-address) |
| No Issues  | [Claimant mailing address](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/non-veteran-mailing-address) |
| No Issues  | [Contact info](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/veteran-phone-and-email) | 
| No Issues  | [Claimant contact info](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/non-veteran-phone-and-email) |
| No Issues  | [Other reasons for request](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/other-reasons) |
| No Issues  | [Other reasons - 3rd party Veteran](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/other-reasons-homeless-third-party-veteran) |
| No Issues  | [Other reasons- 3rd party claimant](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/other-reasons-homeless-third-party-non-veteran) |
| No Issues  | [Financial Hardship evidence](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/evidence-financial-hardship) | 
| No Issues  | [Terminal illness evidence](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/evidence-terminal-illness) |
| No Issues  | [ALS Evidence](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/evidence-als) |
| No Issues  | [VSI Evidence](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/evidence-vsi) |
| No Issues  | [FPOW Dates](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/evidence-pow-confinement) | 
| No Issues  | [FPOW Dates2](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/evidence-pow-confinement-2) |
| No Issues  | [FPOW Evidence](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/evidence-pow-documents) | 
| No Issues  | [Purple heart evidence](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/evidence-medal-award) | 
| No Issues  | [Have you received medical treatment](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/has-received-medical-treatment) | 
| No Issues  | [Medical facilities](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/medical-treatment) | 
| No Issues  | [Medical facilities - 3rd party Veteran](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/medical-treatment-third-party-veteran) |
| No Issues  | [Review](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/review-and-submit) |
| No Issues  | [Confirmation](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/confirmation) | 

### How did axe testing go?

No issues

### Who completed the content zoom and reflow test?

@jeana-adhoc 

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

No issues

### Who completed the keyboard navigation test?

@jeana-adhoc 

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [ ] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

Only one known issue discovered.

**Issue 1** ([Known issue](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/978))
When navigating from page to page in safari with the screen reader on, focus is not defined or announced. The stepper should be in focus on every page load, it is never announced. Note: Focus does load as expected when navigating to 

### Do you have any other results to share?

Not at this time. This was incredibly clean. 

This was early testing - we have some content modifications coming to a few screens and maybe a new page or two. I will add them to this artfact when they are ready.

**Comment by jeana-adhoc** [comment]
Closing this ticket for this sprint. I will do the additional foundational testing on the new pages in their respective tickets and will add any additional findings back into this ticket if necessary.
